README.md

```diff
@@ -231,15 +231,17 @@ spec:
     type = "replicated"
 
     [metadata-client]
-    addresses = ["http://restate:5122/"]
+    addresses = ["http://restate-cluster:5122/"]
```

Comment:
i think this was an oversight, should use the -cluster url as thats the one with publishNotReady turned on, also its a headless whereas the other one is a clusterip